ReactWindows/ReactNative/Views/Scroll/ReactScrollViewManager.cs

```diff
@@ -285,6 +286,8 @@ public override void AddView(ScrollViewer parent, DependencyObject child, int in
             child.SetValue(FrameworkElement.VerticalAlignmentProperty, VerticalAlignment.Top);
             child.SetValue(FrameworkElement.HorizontalAlignmentProperty, HorizontalAlignment.Left);
             parent.Content = child;
+
+            AccessibilityHelper.OnChildAdded(parent, child);
```

Comment:
>OnChildAdded [](start = 32, length = 12)

maybe experiment with adding UpdateAccessibilityOnAdded/RemovedView to base class, calling this helper in base, and overriding it in our Span special case or in soem future custom view managers. Maybe prototype this and show Eric if he will have any problems with this approach?

Reply:
:+1:, just had the same comment in ReactSplitViewManager.

---
In reply to: [169486296](https://github.com/Microsoft/react-native-windows/pull/1645#discussion_r169486296) [](ancestors = 169486296)

Reply:
You'd have to use explicit interface methods to avoid the requirement that the derived class call the base method for these cases.

---
In reply to: [170231013](https://github.com/Microsoft/react-native-windows/pull/1645#discussion_r170231013) [](ancestors = 170231013,169486296)